mypy/literals.py

```diff
@@ -246,3 +248,66 @@ def visit_temp_node(self, e: TempNode) -> None:
 
 
 _hasher: Final = _Hasher()
+
+
```

Comment:
The name `literal` of this `literals.py` file is different from the `Literal` in typing, which is an old and annoying conflict. I am not sure whether they should put in same file.

Reply:
@JukkaL Any suggestions on the new name for the old part?

Reply:
Perhaps `simple_expr.py`? Or even `literal_expr.py`, though I don't think that helps much.